### PR TITLE
ADD: footer links

### PR DESCRIPTION
**Pull request by co012**
Changes:
- Added Terms of Service
- Added Privacy Policy
- Improved social links
Screenshot 2022-09-21 at 15 04 11

Screenshot 2022-09-22 at 12 45 04

### CODE REVIEW

**Comment by alejomendoza** [comment]
Text should be white, sorry about that, I messed up.

File: packages/app/navigation/native/index.tsx [hunk 1]
```diff
@@ -95,6 +98,20 @@ export function NativeNavigation() {
...
+              title: "Terms of Service",
+            }}
+          />
+          <Stack.Screen
+            name="privacy"
+            component={PrivacyScreen}
+            options={{
+              title: "Privacy Policy",
+            }}
+          />
```
Comment (jmysliv) [inline packages/app/navigation/native/index.tsx:114]: I guess for this two, we would like to show a header so users can go back.

File: packages/app/features/legal/termsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,441 @@
+import Navbar from "app/ui/navbar";
+import { SafeAreaView } from "app/design-system/safe-area-view";
+import { ScrollView } from "app/design-system/ScrollView";
+import { Text } from "app/design-system";
+
+export default function TermsScreen() {
+  return (
+    <SafeAreaView className="flex flex-1 bg-blue-dark max-h-[100vh]">
+      <Navbar />
```
Comment (jmysliv) [inline packages/app/features/legal/termsScreen.tsx:9]: I guess we don't need a navbar for mobile.

File: packages/app/features/legal/privacyScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,189 @@
+import { ScrollView } from "app/design-system/ScrollView";
+import { Text } from "app/design-system";
+import Navbar from "app/ui/navbar";
+import { SafeAreaView } from "app/design-system/safe-area-view";
+
+export default function PrivacyScreen() {
+  return (
+    <SafeAreaView className="flex flex-1 bg-blue-dark max-h-[100vh]">
+      <Navbar />
```
Comment (jmysliv) [inline packages/app/features/legal/privacyScreen.tsx:9]: same

File: packages/app/ui/socialLinks.tsx [hunk 4]
```diff
@@ -1,59 +1,44 @@
-import { View } from "app/design-system";
-import { A } from "dripsy";
-import Icon from "@expo/vector-icons/MaterialCommunityIcons";
```
Comment (jmysliv) [inline packages/app/ui/socialLinks.tsx:3]: can we remove this package from the dependencies?

Comment (co012) [reply packages/app/ui/socialLinks.tsx:3]: Still used by dashboard tabbar

Comment (jmysliv) [reply packages/app/ui/socialLinks.tsx:3]: could you replace it there?

**Review by jmysliv — APPROVED** [review]

**Pull request merged**